force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,9 +133,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
-    }
+        if (a==null) {
+            return 'Null';
+
+        } else if (a==''){
+            return 'Empty';
 
+        } else if (!a.isNumeric()) {
```

Comment:
This works but also check if it's empty or the exact characters you are looking for